addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -85,9 +85,17 @@ export class DiscussCoreCommon {
                     model: "discuss.channel",
                     id: payload.res_id,
                 });
-                const message = this.messageService.createTransient(
-                    Object.assign(payload, { body: markup(payload.body) })
-                );
+                const { body, res_id, model } = payload;
+                const lastMessageId = this.messageService.getLastMessageId();
+                const message = this.store.Message.insert({
+                    author: this.store.odoobot,
+                    body: markup(body),
+                    id: lastMessageId + 0.01,
+                    is_note: true,
+                    is_transient: true,
+                    res_id,
+                    model,
+                });
```

Comment:
`createTransient` was only used one: removed the function and put its content here